.eslintrc.json

```diff
@@ -10,6 +10,7 @@
   "rules": {
     "comma-dangle": "off",
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/no-array-index-key": "off",
```

Comment:
I'd prefer to turn this rule off just in the offending line rather than across the whole app.

Could we avoid turning this rule off at all by using the actual chunkindex instead of the array index?

Reply:
Updated this.